src/plugins/orangehrmCorporateDirectoryPlugin/config/routes.yaml

```
@@ -1,4 +1,25 @@
-directory_module_under_development: # remove this
-  path: /directory/{screen}
-  controller: OrangeHRM\Core\Controller\Common\ModuleUnderDevelopmentController::handle
+apiv2_corporate_directory_employee:
+  path: /api/v2/corporate-directory/employees/{empNumber}
```

```suggestion
  path: /api/v2/directory/employees/{empNumber}
```
shall we refactor like this? it will match with module name.

---

done